gtest-parallel

```diff
@@ -360,6 +381,7 @@ def run_job((command, job_id, test)):
   with tempfile.NamedTemporaryFile(dir=options.output_dir, delete=False) as log:
     sub = subprocess.Popen(command + ['--gtest_filter=' + test] +
                              ['--gtest_color=' + options.gtest_color],
+                           env=test_env,
```

Comment:
This confuses me, why is the test environment modified here? Do the binaries pick up GTEST_SHARD.. otherwise?